### ISSUE TITLE
ext.php comment is wrong

### ISSUE THREAD

**Issue by iMattPro** [nickv-phpbb/howto-convert-phpbb30mod-to-phpbb31ext#4]
For the ext.php you have the comment:

```
// this file is not really needed, when empty it can be ommitted
// however you can override the default methods and add custom
// installation logic
```

This is incorrect. The ext.php is absolutely needed. Without it, the extension manager does not recognize the extension, and it will not appear in the extension manager.

**Comment by nickvergessen** [comment]
@VSEphpbb right, I guess it was different before.
The manual is quite old, I'm not sure its even correct anymore in other places aswell

**Comment by nickvergessen** [comment]
Fixed with: phpbb/phpbb#2642
